style.css

```diff
@@ -361,3 +385,157 @@ textarea {
   margin-bottom: 8px;
   border-radius: 15%;
 }
+
+@media screen and (min-width: 414px) {
+  #sct1 .para {
+    width: 100%;
+    max-width: 448px;
+  }
+
+  #sct1 {
+    max-width: 1023px;
+  }
+
+  .skills {
+    max-width: 427px;
+  }
+
+  .join {
+    flex-direction: row;
+    justify-content: space-between;
+  }
+
+  .btn-work {
+    margin-top: 0;
+  }
+
+  .btn-work:first-child {
+    display: flex;
+  }
+
+  .titles h1 {
+    font-size: 70px;
+    font-weight: 700;
+  }
+
+  .titles h2 {
+    font-size: 40px;
+    font-weight: 500;
+  }
+
+  .skills::after {
+    width: 100%;
+  }
+
+  ul.competences:first-child {
+    margin-left: 0;
+  }
+
+  .competences { margin-left: 90px; }
+
+  .work-list {
+    max-width: 1300px;
+    justify-content: center;
+  }
+
+  .work-list-item {
+    flex-direction: row;
+    flex-wrap: wrap;
+  }
+
+  .work-img {
+    width: 100%;
+  }
+
+  .work-list-item:first-child {
+    max-width: 1200px;
+    flex-wrap: wrap;
+    justify-content: flex-start;
+  }
+
+  .contact-us {
+    font-size: 40px;
+    font-weight: 700;
+  }
+
+  .see-project {
+    font-size: 15px;
+    width: 100px;
+  }
+}
+
+@media screen and (min-width: 450px) {
+  .ftr-join::after {
+    display: none;
+  }
+
+  .ftr-join {
+    flex-direction: row;
+    max-width: 421px;
+    margin-bottom: 21px;
+    height: 40px;
+  }
+
+  .my-mail { margin-top: 0; }
+}
+
+@media screen and (max-width: 500px) {
+  form div {
+    flex-direction: column !important;
+  }
+
+  .name-email {
+    width: 100%;
+  }
+}
+
+@media screen and (min-width: 708px) {
+  .name-email {
+    max-width: 329px;
+  }
+
+  form div {
+    justify-content: space-between;
+  }
+}
+
+@media screen and (min-width: 900px) {
+  .btn {
```

Comment:
- [ ] You implemented the desktop version well. However, the value for the min-width property in desktop media queries should be in the range of **992px - 1100px**. Breakpoints represent a subset of common device sizes and viewport dimensions and this range provides a strong and consistent foundation to build on for nearly any desktop screen. Kindly use any value from the aforementioned range for this media query.

Reply:
I understand. But the problem is not to set a specific breakpoint but to make your site responsive so that, when you switch to another device it is still responsive.